distribution/docs/src/main/resources/_contents/_processing-framework/processing-framework-intro-contents.adoc

```diff
@@ -0,0 +1,72 @@
+[NOTE]
+====
+This code is experimental. While this interface is functional and tested, it may change or be removed in a future version of the library.
+====
+
+The *Asynchronous Processing Framework* is a way to run plugins asynchronously. Generally, plugins that take a significant amount of processing time and whose
+results are not immediately required are good candidates for being asynchronously processed. A *Processing Framework* can either be run on the local or
+remote system. Once the *Processing Framework* finishes processing incoming requests, it submits its results back to the Catalog. The type of plugins that a *Processing Framework*
+runs are the *Post-Process Plugins*. These plugins are run after the *Post-Ingest Plugins* and after all metacards have been ingested into the Catalog. This feature is uninstalled by default.
+
+[WARNING]
+====
+The *Processing Framework* does not support partial updates to the Catalog. This means that if any changes are made to a metacard in the Catalog between the time
+asynchronous processing starts and ends, those changes will be overridden by the *ProcessingFramework* updates sent back to the Catalog. This feature should be used with caution.
+====
+
+=== The Asynchronous Processing Framework API Interfaces
```

Comment:
✏️ Given the number of different interfaces and classes involved in this, should we ask @ricklarsen to create a figure (e.g., class diagram, flow diagram, etc.) that shows how all these things relate to one another?